src/mongocxx/private/index_view.hh

```diff
@@ -134,6 +133,25 @@ class index_view::impl {
             opts_doc.append(bsoncxx::builder::concatenate_doc{session->_get_impl().to_document()});
         }
 
+        if (options.commit_quorum()) {
+            auto server_description = libmongoc::client_select_server(
+                _client, true /* for_writes */, nullptr /* read_prefs */, &error);
+            auto is_master = libmongoc::server_description_ismaster(server_description);
+
+            bson_iter_t iter;
+            bson_iter_init_find(&iter, is_master, "maxWireVersion");
```

Comment:
This could return false if the ismaster reply does not contain `maxWireVersion`. I don't think that would realistically happen (if there was no primary available, then `mongoc_client_select_server` would have returned NULL). But I think it is worth checking. If no `maxWireVersion` is available, throw an error.

Reply:
Done